end-to-end-tests/cypress/pages/newTransferProjectWithDecisions.ts

```diff
@@ -0,0 +1,83 @@
+// newTransferProjectWithDecisions.ts
+
+export class NewTransferProjectWithDecisions {
+  visit(url: string) {
```

Comment:
To fit in with other page objects, worth making methods return `this` so we can chain the calls - makes the code a bit cleaner in the tests and makes us consistent. See e.g. `incomingTrustSearch` as an example